crates/compiler/gen_llvm/src/llvm/build_str.rs

```diff
@@ -63,3 +63,19 @@ pub(crate) fn str_equal<'a, 'ctx, 'env>(
         bitcode::STR_EQUAL,
     )
 }
+
+// Gets a pointer to just after the refcount for a list or seamless slice.
+// The value is just after the refcount so that normal lists and seamless slices can share code paths easily.
+pub(crate) fn str_refcount_ptr<'a, 'ctx, 'env>(
```

Comment:
not the best name (the pointer is not to the refcount), but I don't have better suggestions

Reply:
Yeah, was the same issue with lists. The function was originally actually refcount pointer but then I changed it for performance. Maybe just allocation pointer, or original str pointer?